src/main/java/com/ing/data/cassandra/jdbc/CassandraDataSource.java

```diff
@@ -281,6 +299,9 @@ public CassandraConnection getConnection(final String user, final String passwor
         if (this.localDataCenter != null) {
             props.setProperty(TAG_LOCAL_DATACENTER, localDataCenter);
         }
+        if (this.complianceMode != null) {
+            props.setProperty(KEY_COMPLIANCE_MODE, complianceMode);
```

Comment:
You have to use `TAG_COMPLIANCE_MODE` instead of `KEY_COMPLIANCE_MODE`. The `TAG_...` values are used for internal usage and build the configuration map used to instantiate the connection and the `KEY_...` values are used to define the query parameters accepted in the JDBC URL.

Reply:
yes, i've noticed that. but KEY_COMPLIANCE_MODE and TAG_COMPLIANCE_MODE are different (compliancemode vs complianceMode). i think it didnt work with TAG_COMPLIANCE_MODE, but i will double check.
liquibase documenation says it should be "compliancemode" lowercase.

Reply:
right. locally it's correct. i will fix, thx

Reply:
done